aspnetcore/signalr/scale.md

```diff
@@ -104,14 +104,82 @@ The preceding conditions make it likely to hit the 10 connection limit on a clie
 
 ## Linux with Nginx
 
-Set the proxy's `Connection` and `Upgrade` headers to the following for SignalR WebSockets:
+The following contains the minimum required settings to enable WebSockets, ServerSentEvents, and LongPolling for SignalR:
 
 ```nginx
-proxy_set_header Upgrade $http_upgrade;
-proxy_set_header Connection $connection_upgrade;
+http {
+  map $http_upgrade $connection_upgrade {
+    default Upgrade;
+    '' close;
+  }
+
+  server {
+    listen 80;
+    server_name example.com *.example.com;
+
+    # Configure the SignalR Endpoint
+    location /hubroute {
+      # App server url
+      proxy_pass http://localhost:5000;
+
+      # Configuration for WebSockets
+      proxy_set_header Upgrade $http_upgrade;
+      proxy_set_header Connection $connection_upgrade;
+      proxy_cache off;
+
+      # Configuration for ServerSentEvents
+      proxy_buffering off;
+
+      # Configuration for LongPolling
+      proxy_read_timeout 100s;
+
+      proxy_set_header Host $host;
+      proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+      proxy_set_header X-Forwarded-Proto $scheme;
+    }
+  }
+}
 ```
 
-For more information, see [NGINX as a WebSocket Proxy](https://www.nginx.com/blog/websocket-nginx/).
+When multiple backend servers are used, sticky sessions must be added to prevent SignalR connections from switching servers when connecting. There are multiple ways to add sticky sessions in Nginx. Two approaches are shown below depending on what you have available.
+
+The following is added in addition to the previous configuration. `backend` in the following example is the name of the group of servers:
+
+With Nginx Open Source, use `ip_hash` to route connections to a server based on the clients IP address.
```

Comment:
```suggestion
With [Nginx Open Source](https://nginx.org/en/), use `ip_hash` to route connections to a server based on the client's IP address:
```

Reply:
btw - the "en" localization is required for their doc to load. **_Shame on them!_** :smile: lol